source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -66,7 +66,8 @@ absl::string_view Extractor::extractValue(absl::string_view value) const {
   if (std::regex_match(value.begin(), value.end(), regex_result,
                        extract_regex_)) {
     if (group_ >= regex_result.size()) {
-      // TODO: this indicates user misconfiguration... log or stat?
+      ASSERT("no such group in the regex");
+      //ENVOY_STREAM_LOG(debug, "invalid group specified for regex", callbacks_);
```

Comment:
Why is this commented out? you can inherit from `Logger::Instance` in these classes to get access. In general I feel that these filters would benefit from more logging

Reply:
fixed